assets/components/directDebit/directDebitReducer.js

```diff
@@ -8,7 +8,7 @@ import type { Action } from './directDebitActions';
 
 export type DirectDebitState = {
   isPopUpOpen: boolean,
-  bankSortCode: string,
+  sortCodeArray: Array<string>,
   bankAccountNumber: string,
```

Comment:
I would rename `bankAccountNumber` to `accountNumber` so in that way it will work the destructuring piece in the Actions file.

Reply:
Done